scripts/cluster/setup_master_node.sh

```diff
@@ -48,7 +48,7 @@ export PATH=$PATH:$ROOT/istio-1.12.5/bin
 sudo sh -c  "echo 'export PATH=\$PATH:$ROOT/istio-1.12.5/bin' >> /etc/profile"
 istioctl install -y -f $ROOT/configs/istio/istio-minimal-operator.yaml
```

Comment:
Please do update Istio as well if you already updated K8s and Knative. 
You need to find the proper version of K8s, Knative, and Istio that all support each other. 

Here is the compatibility table:
- https://istio.io/latest/docs/releases/supported-releases/

It also looks like the deployment method for Istio has changed, hence you should adapt the script:
- https://github.com/knative-sandbox/net-istio/releases
- https://knative.dev/docs/install/installing-istio/#basic-installation-with-istioctl